hints.md

```diff
@@ -11574,7 +11622,7 @@ a ?~ b
 <td>
 LHS:
 <code>
-a & (mapped %~ b)
+(mapped %~ b) a
```

Comment:
Don't know why this is happening

Reply:
Odd. Which GHC version are you on? I'm using 9.0.1